modules/commons/src/main/resources/schemas/neurosciencegraph/commons/protocol/v0.1.5.json

```diff
@@ -0,0 +1,69 @@
+{
+  "@context": [
+    "{{base}}/contexts/neurosciencegraph/core/schema/v0.1.0",
+    {
+      "this": "{{base}}/schemas/neurosciencegraph/commons/protocol/v0.1.4/shapes/"
+    }
+  ],
+  "@type": "nxv:Schema",
+  "imports": [
+    "{{base}}/schemas/neurosciencegraph/commons/entity/v0.1.0",
+    "{{base}}/schemas/nexus/schemaorgsh/identifier/v0.1.0",
+    "{{base}}/schemas/neurosciencegraph/commons/organization/v0.1.0",
+    "{{base}}/schemas/neurosciencegraph/commons/person/v0.1.0"
+  ],
+  "shapes": [
+    {
+      "@id": "this:ProtocolShape",
+      "@type": "sh:NodeShape",
+      "label": "Protocol shape definition.",
+      "nodeKind": "sh:BlankNodeOrIRI",
+      "and": [
+        {
+          "node": "{{base}}/schemas/neurosciencegraph/commons/entity/v0.1.0/shapes/EntityShape"
+        },
+        {
+          "path": "schema:identifier",
+          "name": "Identifier",
+          "description": "The identifier of protocol such as a DOI, ISBN,...",
+          "node": "{{base}}/schemas/nexus/schemaorgsh/identifier/v0.1.0/shapes/IdentifierShape"
+        },
+        {
+          "path": "schema:datePublished",
+          "name": "date published",
+          "description": "Publishing date of protocol.",
+          "datatype": "xsd:dateTime",
+          "maxCount": 1
+        },
+        {
+          "path": "schema:author",
+          "name": "Author",
+          "description": "Author of the protocol",
+          "xone": [
+            {
+              "node": "{{base}}/schemas/neurosciencegraph/commons/person/v0.1.0/shapes/PersonShape"
+            },
+            {
+              "node": "{{base}}/schemas/neurosciencegraph/commons/organization/v0.1.0/shapes/OrganizationShape"
+            }
+          ]
+        },
+        {
+          "path": "dcterms:hasPart",
+          "name": "Has part",
+          "description": "Sub-protocols of the protocol.",
+          "class": "nsg:Protocol"
```

Comment:
I would recommend to remove the class constraint until we get shape recursion right. You could add a seeAlso property can could link to nsg:Protocol.

Reply:
Done.